libs/core/src/lib/notification/notification/notification.component.ts

```diff
@@ -90,14 +96,23 @@ export class NotificationComponent extends AbstractFdNgxClass implements OnInit,
         private cdRef: ChangeDetectorRef,
         private _router: Router,
         @Optional() private notificationConfig: NotificationConfig,
-        @Optional() private notificationRef: NotificationRef
+        @Optional() private notificationRef: NotificationRef,
+        @Optional() private _rtlService: RtlService
     ) {
         super(elRef);
         this._setNotificationConfig(notificationConfig);
     }
 
     /** @hidden */
     ngOnInit(): void {
+
+        this._subscriptions.add(
+            this._rtlService?.rtl.subscribe(isRtl => {
+                this._dir = isRtl ? 'rtl' : 'ltr';
+                this.cdRef.markForCheck();
```

Comment:
Do we have some code style guide how to use subscription? 
I mean, the subscription and the success handler must be on the same line, or vice versa, the subscription must always start on a new line, and each handler in the subscription?

Reply:
I don't think that we have such code style. It was copy-pasted from different component, and as I see, majority of components use the same code style.

Reply:
cdRef is also private and should be prefixed with `_`